packages/components/time-field/src/time-field.stories.ts

```diff
@@ -0,0 +1,156 @@
+import '@sl-design-system/form/register.js';
+import { type Meta, type StoryObj } from '@storybook/web-components-vite';
+import { html, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import '../register.js';
+import { type TimeField } from './time-field.js';
+
+type Props = Pick<
+  TimeField,
+  | 'disabled'
+  | 'hourStep'
+  | 'locale'
+  | 'max'
+  | 'min'
+  | 'minuteStep'
+  | 'placeholder'
+  | 'readonly'
+  | 'required'
+  | 'start'
+  | 'value'
+> & {
+  hint?: string;
+  label?: string;
+  reportValidity?: boolean;
+};
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Form/Time field',
+  tags: ['draft'],
+  args: {
+    disabled: false,
+    label: 'Time',
+    placeholder: 'Select time',
+    readonly: false,
+    required: false,
+    value: ''
+  },
+  argTypes: {
+    hint: { table: { disable: true } },
+    label: { table: { disable: true } },
+    locale: {
+      control: 'inline-radio',
+      options: ['de', 'en-GB', 'es', 'fi', 'fr', 'it', 'nl', 'nl-BE', 'no', 'pl', 'sv']
+    },
+    reportValidity: { table: { disable: true } }
+  },
+  render: ({
+    disabled,
+    hint,
+    hourStep,
+    label,
+    locale,
+    max,
+    min,
+    minuteStep,
+    placeholder,
+    readonly,
+    reportValidity,
+    required,
+    start,
+    value
+  }) => {
+    const onClick = (event: Event & { target: HTMLElement }): void => {
+      event.target.closest('sl-form')?.reportValidity();
+    };
+
+    return html`
+      <sl-form>
+        <sl-form-field .hint=${hint} .label=${label}>
+          <sl-time-field
+            ?disabled=${disabled}
+            hour-step=${ifDefined(hourStep)}
+            locale=${ifDefined(locale)}
+            max=${ifDefined(max)}
+            min=${ifDefined(min)}
+            minute-step=${ifDefined(minuteStep)}
+            placeholder=${ifDefined(placeholder)}
+            ?readonly=${readonly}
+            ?required=${required}
+            start=${ifDefined(start)}
+            value=${ifDefined(value)}
+          ></sl-time-field>
+        </sl-form-field>
+        ${reportValidity
+          ? html`
+              <sl-button-bar>
+                <sl-button @click=${onClick}>Report validity</sl-button>
+              </sl-button-bar>
+            `
+          : nothing}
+      </sl-form>
+    `;
+  }
+} satisfies Meta<Props>;
+
+export const Basic: Story = {};
+
+export const Disabled: Story = {
+  args: {
+    disabled: true
+  }
+};
+
+export const Finnish: Story = {
+  args: {
+    hint: 'In Finnish (fi) the time separator is a dot (.)',
+    locale: 'fi',
+    value: '13.30'
+  }
+};
+
+export const MinMax: Story = {
+  args: {
+    hint: 'The allowed time range is between 08:00 and 18:00',
+    min: '08:00',
+    max: '18:00',
+    start: '09:00'
+  }
+};
+
+export const Readonly: Story = {
+  args: {
+    readonly: true,
+    value: '13:30'
+  }
+};
+
+export const Required: Story = {
```

Comment:
In this case, the field should be invalid right? It's not set as invalid (even when clicked reportValidity) and invalid state is not being shown
<img width="1544" height="571" alt="image" src="https://github.com/user-attachments/assets/8f939f74-72ea-46f8-bef2-17eaa791e6d4" />

Reply:
The `<input>` can be valid while the time-field isn't.